apps/website/screens/common/MainContent.tsx

```diff
@@ -0,0 +1,64 @@
+import { useToast } from "@dxc-technology/halstack-react";
+import { ReactNode, useMemo, useState, useEffect } from "react";
+import styled from "styled-components";
+import { responsiveSizes } from "./variables";
+import { useRouter } from "next/router";
+
+const MainContainer = styled.div`
+  margin: 80px 0;
+  max-width: 1124px;
+  margin-inline: max(5%, 50% - 1124px/2);
+
+  @media (max-width: ${responsiveSizes.laptop}px) {
+    margin: 80px 32px;
+  }
+`;
+
+const MainContent = ({ children }: { children: ReactNode }) => {
+  const toast = useToast();
+  const router = useRouter();
+  const pathVersion = useMemo(
+    () =>
+      process.env.NEXT_PUBLIC_SITE_VERSION === "next" || process.env.NODE_ENV === "development"
+        ? 0
+        : parseInt(process.env.NEXT_PUBLIC_SITE_VERSION?.split(".")[0], 10),
+
+    []
+  );
+  const [latestRelease, setLatestRelease] = useState<number>(null);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await fetch("https://registry.npmjs.org/@dxc-technology/halstack-react/");
+        const data = await response.json();
+        const latestRelease = parseInt(data["dist-tags"].latest.split(".")[0], 10);
+        setLatestRelease(latestRelease);
+      } catch (error) {
+        console.error("Error fetching version:", error);
+      }
+    })();
+  }, []);
+
+  useEffect(() => {
+    if (latestRelease > pathVersion) {
+      toast.info({
+        message: `Halstack ${latestRelease} is now available!`,
+        action: {
+          label: "Learn more",
+          onClick: () => {
+            if (window) {
+              const currentUrl = window.location.href;
+              const newUrl = currentUrl.replace(/halstack\/\d+\//, `halstack/${latestRelease}/`);
+              window.location.href = newUrl;
+            }
+          },
+        },
+      });
+    }
+  }, [latestRelease, toast]);
```

Comment:
`pathVersion` should probably be added to the dependency array

Reply:
Totally true!